Bottom: a076655dffcfce39473beebb05ac33590b4949f2
Top:    e24416be052a4e58d8a86686440c85d9f5b4ca1e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index bb39aa7..d2a75d7 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1219,8 +1219,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
